### PR TITLE
Component refactor

### PR DESCRIPTION
**Pull request by JakeAdmire**
## Changes Made:
- Refactored `Thunk` naming convention 
- Built dynamic components within `styleIndex.js`
- Built out styling for `RoutineDetails` component
- Implemented `styleIndex` dynamic components throughout application

### Relevant Screenshots:
N/A

### Message for reviewer:
Please review these changes

### Testing complete?
- [ ] Yes
- [x] Not yet
- [ ] N/A

### Final checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by JakeAdmire** [comment]
Closes Issue #67

File: src/Components/Container/Container.js [hunk 1]
```diff
@@ -3,12 +3,14 @@ import { connect } from 'react-redux';
 import Dimensions from 'Dimensions';
 import { Text, View, StyleSheet, Image } from 'react-native';
 import { Card, Icon, ListItem, Button } from 'react-native-elements';
+// 
 import { RalewayText, RalewayBoldText } from '../../Utilities/RalewayText';
 import { setSelectedRoutine } from '../../Actions';
-import * as palette from '../../Utilities/styleIndex';
-import { deleteRoutineThunk } from '../../Thunks/deleteRoutineThunk'
+import { palette, flexibleButton } from '../../Utilities/styleIndex';
+import { deleteRoutine } from '../../Thunks/deleteRoutine'
```
Comment (ericfitzsimons451) [inline src/Components/Container/Container.js:10]: Better name.  Good idea

File: src/Utilities/styleIndex.js [hunk 2]
```diff
@@ -1,7 +1,84 @@
...
+  shadowColor: "#000",
+  shadowOffset: { width: 0, height: 6, },
+  shadowOpacity: 0.37,
+  shadowRadius: 7.49,
+  // 
+  elevation: 12
+}
+
+
+export const welcomeText = (name) => {
```
Comment (ericfitzsimons451) [inline src/Utilities/styleIndex.js:30]: Fantastic.  This is the essence of component-based code!  Very good work here, my friend!

File: src/Components/Container/Container.js [hunk 3]
```diff
@@ -3,12 +3,14 @@ import { connect } from 'react-redux';
 import Dimensions from 'Dimensions';
 import { Text, View, StyleSheet, Image } from 'react-native';
 import { Card, Icon, ListItem, Button } from 'react-native-elements';
+//
```
Comment (ericfitzsimons451) [inline src/Components/Container/Container.js:6]: Glad you told me why you separate the package imports from the codebase imports.  Good for dev empathy.

**Comment by JakeAdmire** [comment]
Closes Issue #61

**Pull request merged**